fluentui_controls/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
this file has not been part of any module. Check once, if its required to have it here.

Reply:
This is required as the build file (generated) should not show up on git as new files